src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
 document.addEventListener('click', e => {
-  // write code here
+  const item = e.target;
+
+  if (item !== wall) {
```

Comment:
Replace the condition with 'item !== wall' to 'item === wall' and remove the 'return' statement to reduce the indentation level of the following code.